### PR TITLE
[TASK][27659] Update homepage

### PR DESCRIPTION
**Pull request by thuyptsun**
https://edu-redmine.sun-asterisk.vn/issues/27659
Screenshot from 2020-08-21 14-03-31
Screenshot from 2020-08-21 14-03-38
Screenshot from 2020-08-21 14-03-47

### CODE REVIEW

**Review by vinhcnt — APPROVED** [review]

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class CategoriesController < ApplicationController
+  def index
+    @categories = Category.all
```
Comment (hieplt-0914) [inline app/controllers/categories_controller.rb]: Thêm phân trang vào em

File: app/views/categories/_category.html.erb [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  <%= link_to categories_path(category) do %>
     <div class="popu-category-box bg-light rounded text-center p-4">
       <div class="popu-category-icon mb-3">
         <i class="fa fa-briefcase rounded-pill" aria-hidden="true"></i>
       </div>
       <div class="popu-category-content">
-        <h5 class="mb-2 text-dark title">Category Name</h5>
-        <p class="text-success mb-0 rounded">123 Jobs</p>
+        <h5><%= link_to category.title, categories_path(category), class: "mb-2 text-dark title" %></h5>
+        <p class="text-success mb-0 rounded"><%= category.job_posts.count %><%= t(".jobs") %></p>
```
Comment (hieplt-0914) [inline app/views/categories/_category.html.erb]: size

Comment (hieplt-0914) [inline app/views/categories/_category.html.erb]: ()

File: app/models/job_post.rb [hunk 3]
```diff
@@ -4,4 +4,7 @@ class JobPost < ApplicationRecord
   belongs_to :unit
 
   delegate :title, to: :category, prefix: true, allow_nil: true
+  delegate :name, to: :unit, prefix: true, allow_nil: true
+
+  scope :home_recent_jobs, ->{order(created_at: :desc).limit(Settings.job)}
```
Comment (hieplt-0914) [inline app/models/job_post.rb]:
```suggestion
  scope :home_recent_jobs, ->{order(created_at: :desc).limit Settings.job}
```

**Pull request merged**